scripts/deploy-operator.sh

```diff
@@ -45,8 +45,21 @@ fi
 		echo "ERROR: CSV not deployed. Operator deployment failed -- interrupting tests"
 		exit 1
 	fi
+COMMENT2
 
-'
-
-# deploy single namespace operator nginx in namespace nginx-ops
+# Deploy single namespace operator nginx in namespace nginx-ops
 oc apply --filename ./test-target/operator-single-install-mode.yaml
+
+# Wait until operator pods is in running state
+OPERATOR_NS="nginx-ops"
+while true; do
```

Comment:
Can this have a timeout/limit value?  I can see this might get "stuck" forever looping if something goes wrong.

Reply:
Yeah, running for 10 iterations.